back/app/services/content_image_service.rb

```diff
@@ -0,0 +1,148 @@
+class ContentImageService
+  class DecodingError < StandardError
+    def initialize(options = {})
+      super
+      @parse_errors = options.fetch :parse_errors, nil
```

Comment:
Given the `nil` default in `fetch`, this code is equivalent and shorter:
```suggestion
      @parse_errors = options[:parse_errors]
```